build-binary.sh

```diff
@@ -82,7 +87,7 @@ if [ -f multidist.buildinfo ]; then
 		fi
 
 		# Ensure orig tarball exists
-		find "$rootwp" \
+		find "$rootwp/mbuild" \
```

Comment:
Maybe use `../` instead, so that the link stays valid when moving between machines.

```suggestion
		find ../ \
```

Reply:
Done